src/messages.rs

```diff
@@ -33,34 +34,44 @@ pub struct WebDriverError {
     pub stacktrace: Option<String>,
 }
 
+#[derive(Serialize, Default)]
+pub struct Capabilities {
+    alwaysMatch: BTreeMap<String, JsonValue>,
+}
+
 #[derive(Serialize)]
 pub struct NewSessionCmd {
-    capabilities: JsonValue,
+    capabilities: Capabilities,
 }
 
 impl NewSessionCmd {
     pub fn new() -> Self {
+        let mut capabilities: Capabilities = Default::default();
+        capabilities.alwaysMatch.insert("goog:chromeOptions".to_string(), json!({"w3c": true}));
```

Comment:
Can we push `goog:chromeOptions` down into a manual implementation of `Default` on `struct Capabilities`? That seems a little less surprising to me and allows the user to create a session with complete control over capabilities if required.